### DISCUSSION TITLE
What is the best exercise for weight loss ?

### DISCUSSION THREAD

**Discussion by mozuku-works** [NAIST-SE/AttractingContributors#816] [2023 Attraction Contributions]
What is the best exercise for weight loss ? 
It's about time for me to lose weight.
Please let me know about your favorite exercise!!

The link to my repository is as shown below:

https://github.com/mozuku-works/AttractingContributors-exercise

Thanks.

**Comment by ellielo** [comment]
Fitness Zumba is great!
